setup.py

```diff
@@ -12,8 +12,8 @@
     version='0.1.0',
     description='CellML loading and model equation manipulation',
     long_description=readme,
-    author='',
-    author_email='',
+    author=['Asif U Tamuri', 'Sarah M Keating', 'Maurice Hendrix', 'Michael Clerx', 'Jonathan Cooper'],
```

Comment:
Not sure we can do this https://stackoverflow.com/questions/9999829/how-to-specify-multiple-authors-emails-in-setup-py